src/plugins/sync-font.js

```diff
@@ -0,0 +1,16 @@
+export function clientSyncFont(client) {
+	return client.request('font').then((font) => {
+		document.body.style.fontFamily = font.family;
+		document.body.style.fontSize= font.size;
+	});
+}
+
+export function hostSyncFont(host) {
+	host.onRequest('font', () => {
+		const computedStyle = window.getComputedStyle(document.body);
```

Comment:
I don't recall how expensive getComputedStyle is.  It looks like each app would only cause this to be called once?  Or, considering a case where widgets are apps, this could be called multiple times at init-time.  We could optimize this by caching the result of getComputedStyle?  My understanding is that this host plugin instance would be shared if there was more than one app on a page, yes?